### ISSUE TITLE
Address accessibility review feedback

### ISSUE THREAD

**Issue by beefoo** [beefoo/lclabs-jfp24#57]
1. I suggest requesting a waiver for full screen reader support based on the essential exception. When it is ready, I can help you draft that. 
2. Voice control users should be able to use this so I would make a few changes to the buttons on the left. I would add the title to the button instead of the image and then I would make sure the alt text which is serving as the button name matches the title (remove the word tool). 
3. Sighted keyboard users should be able to use this. You’ve already built in a lot of keyboard accessibility but can you add keyboard commands to manipulate images after selected? Something like:
    - Enter to select, 
    - Arrow keys to move around, 
    - Enter or escape to release, 
    - Plus and minus to change selected image size, and
    - An accesskey=R or a similar functionality (Shift+Alt+R) to rotate clockwise                                                                                                                             
    - The keyboard shortcuts would need to be listed somewhere as well, but could be a github page with a link or in the information section. 
4. Voice 
5. Place the breakpoint such that the text “Change background” moves with the buttons when the title of the image is longer (example: The Causeway) or the viewport is smaller.
6. This is usability more than accessibility but the look and location of the Reset button is confusing. It also deletes all the work without warning the user. I suggest a few changes to make this more successful:
    - Reset is typically a circular arrow:   The current new document icon would typically create a new without deleting the existing work. 
    - I suggest moving reset down so that Save is your first icon, followed by undo, redo and then reset. That separates the two opposing buttons and reduces the likelihood of mistakes
    - Finally, I would add a confirmation – do you want to reset this image? Yes/no
